script/cucumber

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 time ruby -S bundle exec cucumber \
-  -f Jekyll::Cucumber::Formatter "$@"
+-f summary "$@"
```

Comment:
It'd be nice to leave the indentation as ln#4 is a continuation of ln#3.. instead of a new expression..